LoadTests/SET_UP_TEST_ENTITIES.sql

```diff
@@ -60,10 +66,11 @@ DO $$
                 (STARTING_ID + INDEX, STARTING_ID + INDEX, 0, '01/10/2020 12:26:44', '01/10/2020 12:26:44', '01/10/2020 12:26:44');
 
                 -- Create test organisations that are NOT linked to test users
+                -- These have the same names as linked Organisations, so they appear in the search results together

```

Comment:
This is because we want to be able to search for any org - linked or not - in the first section of the test, so to simplify that we use the same names so they all come up (with different IDs) when searched!